src/commands/init.ts

```diff
@@ -187,19 +192,45 @@ export default class Init extends Command {
     }
     packageJson.scripts = { ...packageJson.scripts, ...scripts };
 
-    if (initializationType === InitType.freshExpress) {
-      packageJson.devDependencies = { ...packageJson.devDependencies, ...packageJsonData.devDependencies };
-    }
+    await this.addDependencies(packageJsonData.dependencies, false, packageJson, flags.projectDir);
+    await this.addDependencies(packageJsonData.devDependencies, true, packageJson, flags.projectDir);
```

Comment:
If this is only used in this file, why not just get rid of the idDev flag and always run it for all dependencies?